agent/src/beerocks/fronthaul_manager/ap_manager/ap_manager_thread.cpp

```diff
@@ -357,6 +354,32 @@ bool ap_manager_thread::handle_cmdu(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_
     }
 
     switch (beerocks_header->action_op()) {
+    case beerocks_message::ACTION_APMANAGER_CONFIGURE: {
+        if (m_ap_manager_configured) {
+            break;
+        }
+
+        LOG(TRACE) << "ACTION_APMANAGER_CONFIGURE";
+
+        auto config = beerocks_header->addClass<beerocks_message::cACTION_APMANAGER_CONFIGURE>();
+        if (!config) {
+            LOG(ERROR) << "addClass cACTION_APMANAGER_CONFIGURE failed";
+            return false;
+        }
+
+        low_filter   = config->iface_filter_low();
+        m_iface_type = config->iface_type();
+        wifi_channel = config->channel();
```

Comment:
AFAICS, this is not used at all, so maybe remove it entirely?

Actually, originally it was used to set acs_enabled, but I don't see that any more? And also I think acs_enabled could be handled better than through the configuration. AFAICS it is only used in two places:
- in dwpal's attach function, but only for error messages;
- in handle_hostap_attached(), to choose between read_acs_report or read_preferred_channels.

Note that *doesn't* seem to be used to actually configure bwl into ACS, so I guess there's a paralle configuration knob in the wireless UCI config to enable ACS.

Therefore, it would make a whole lot more sense to me to handle the ACS option completely within bwl. Indeed, whether ACS is enabled or not, we should simply be able to read the channel preferences from hostapd - the difference is that hostapd will return different preference values.

Obviously, that's too big to include in this PR though...

Reply:
You are right, I didn't notice that they are not being used anymore.
I will remove `low_filter` and `wifi_channel`.
m_iface_type is actually do needed, and it should be set on base_wlan_hal() constructor. For some reason, it uses everywhere `IfaceType::Intel` although it not could be another platform.
On the other hand, why we need to set it from outside so we could use get_iface_type() and send it back to ap_manager when son_slave already has this information.

I will remove the above from the config message and from the class. But I'll leave the `acs_enable`. Also, I've just noticed that I forgot to set this member.